python/dlisio/plumbing/frame.py

```diff
@@ -396,19 +390,8 @@ def curves(self):
         DataFrame
 
         >>> curves = frame.curves()
-        >>> if frame.index_type:
-        >>>     indexname  = curves.dtype.names[0]
-        >>>     curvenames = curves.dtype.names[1:]
-        >>>     pdcurves = pd.DataFrame(curves[list(curvenames)], index=curves[indexname])
-        >>>     pdcurves.index.name = indexname
-        >>> else:
-        >>>     pdcurves = pd.DataFrame(curves)
-
-        Let's walk through this. Firstly, check that *index_type* is not None,
-        if it is, there is no index channel to set. Secondly, note how the
-        curvenames are all stored in the array itself. Then all that is needed
-        is to extract the first name as index and let the rest be added to the
-        DataFrame as normal Channels.
+        >>> pdcurves = pd.DataFrame(curves[curves], index=curves[f.index])
```

Comment:
Do I understand correctly that index channel (or frameno) will appear twice? As index as as part of the data?